ui/components-react/src/components-react/toolbar/PopupItem.scss

```diff
@@ -14,33 +14,37 @@
   $arrow-shadow: $icon-shadow;
   position: relative;
 
-  >* {
+  > * {
     &:not(.components-triangle) {
       height: 100%;
       width: 100%;
     }
   }
 
-  >.components-triangle {
+  > .components-triangle {
     border-radius: 0;
     position: absolute;
     right: $margin-from-triangle-to-stroke;
     bottom: $margin-from-triangle-to-stroke;
     filter: $arrow-shadow;
     pointer-events: none;
 
-    @include triangle-bottom-right($width: $triangle-width, $height: $triangle-height, $color: $buic-icon-color);
+    @include triangle-bottom-right(
+      $width: $triangle-width,
+      $height: $triangle-height,
+      $color: $buic-icon-color
+    );
   }
 
-  &:active>.components-triangle {
+  &:active > .components-triangle {
     filter: $no-shadow;
 
     @include triangle-bottom-right-color($pressed-icon-color);
   }
 
   &.components-active {
     &:not(:active):not(.components-disabled) {
-      >.components-triangle {
+      > .components-triangle {
```

Comment:
was these spaces added on purpose... I think they mean different things with and without spaces?

Reply:
Removed! Thanks.